Update link to additional profile configurations (#403)
fishtown-analytics/docs.getdbt.com: website/docs/dbt-cli/configure-your-profile.md

## What goes in my `profiles.yml` file?
In your `profiles.yml` file, you can store as many profiles as you need. Typically, you would have one profile for each warehouse you use – for most organizations, this means you would only have one profile.

> You can also configure some advanced dbt options in your `profiles.yml` file, see [Additional profile configurations](configure-your-profile/#additional-profile-configuration).

Reviewer: I think this is not quite what we want it to say today, now that we have a reference section (which needs to be finished)

```suggestion
You can also configure some advanced dbt options in your `profiles.yml` file, see [the reference docs for `profiles.yml` files](reference/profiles.yml.md).
```
